src/AspNetCore/Src/aspnetcoreconfig.cxx

```diff
@@ -376,6 +376,7 @@ ASPNETCORE_CONFIG::Populate(
         strExpandedEnvValue.Reset();
         pEnvVar->Release();
         pEnvVar = NULL;
+        pEntry->Dereference();
```

Comment:
Is this related to the fix or is it unrelated.

Reply:
this fixes a memory leak